### ISSUE TITLE
How to add libraries like andare to a lumo build?

### ISSUE THREAD

**Issue by abhi18av** [anmonteiro/lumo#487]
Hi everyone,

I was wondering about whether it's possible to add other libraries to `lumo` during the build process. I'd like to have custom build which includes all the libraries which I wish to get better at. I tried to find the specific file in the repo which lists the included libs such as `core.spec.alpha` but couldn't find what I was looking for.

Could you please help me out?

**Comment by arichiardi** [comment]
Do you mean to bundle them in the lumo executable?

**Comment by abhi18av** [comment]
Yes, I'd like to have a certain set of libraries right inside lumo to be able to practice and help others learn faster as well.
________________________________

**Comment by arichiardi** [comment]
I am not sure it is worth the effort but you would need to fork lumo and tweak the `build.boot` first then the node nice wrapper for that.

If you want to *use* libraries and not embed them you could just follow this:

https://github.com/anmonteiro/lumo/wiki/Run

I hope this clarifies a bit my question as it seems that the latter approach is easier.

**Comment by abhi18av** [comment]
Hi @arichiardi , thanks for your quick reverts ;) 

Actually, I just would like to have people install a single binary to get started with `clojurescript` and through this they could have a good playground without all the other incantations which come with other tools. Simpler things facilitate learning faster.

So, if I just add ` [andare "0.10.0"]` to the `build.boot` that'd enough to have this available in the main binary right?

**Comment by arichiardi** [comment]
Nope you would need to tweak the part that compiles and embeds it in node. I personally have never done that.

I see now your problem and I can maybe propose another easier solution, which is to give students a script that wraps lumo and sets things up :smile:

Btw if you will go for that (big) attempt, please paste here the fork/branch name cause I would like to take a look at well :wink:

**Comment by abhi18av** [comment]
Yes, the `script` oriented way is my fallback solution for sure 😄  But I'd definitely like to try how it works in practice as well

Here's my fork/branch

https://github.com/abhi18av/lumo/tree/abhi18av-with-andare
